milner_desvars/gazebo.md

```diff
@@ -0,0 +1,71 @@
+#Notes
+
+##Client and server separation
+
+When you run Gazebo you are actually running to different executables. In one hand "gzserver" and in the other "gzclient".
+
+The gzserver executable runs the physics update-loop and sensor data generation. This is the core of Gazebo and it can be used independtly of a graphical interphase. 
+
+The gzclient executable runs a QT based user interface. This application provides a nice visualization of simulation, and convenient controls over various simulation properties.
+
+## World files
+
+Contains the elements of the simulation, robots, sensors, etc. This file is formatted using SDF (Simation Description Format), and typically have a .world extension.
+
+The Gazebo server reads this file to generate and populate a world.
+
+## Model files
+
+A model file uses the same SDF format as world files, but should only contain a single <model> ... <model>.The purpose of these files is to facilitate model reuse, and simplify world files.
+
+## Enviroment Variables
+
+Gazebo uses a number of environment variables to locate files, and set up communications between the server and clients. Default values that work for most cases are compiled in. Some enviroment variables are:
+
+GAZEBO_MODE_PATH
```

Comment:
Use lists in Markdown and code for the names. More info [here](https://en.support.wordpress.com/markdown-quick-reference/).